lib/jekyll/algolia/logger.rb

```diff
@@ -40,7 +40,7 @@ def self.verbose(line)
       def self.known_message(message_id, metadata = {})
         file = File.expand_path(
           File.join(
-            File.dirname(__FILE__), '../../..', 'errors', "#{message_id}.txt"
+            __dir__, '../..', 'errors', "#{message_id}.txt"
```

Comment:
@DirtyF is it ok that there is removed `..`?

Reply:
Good catch, it was actually an error